src/clib/xtg/well_mask_shoulder.c

```diff
@@ -0,0 +1,84 @@
+/*
+****************************************************************************************
+*
+* NAME:
+*    well_mask_shoulder.c
+*
+* DESCRIPTION:
+*    Given a TVD or a MD log, and a discrete log, return a mask log
+*    based on distance criteries
+*
+* ARGUMENTS:
+*    lvec           i     MD or TVD log
+*    nvec           i     Number of elements in lvec
+*    inlog          i     Input log
+*    ninlog         i     Number of points in inlog
+*    distance       i     Distance to evaluate
+*
+* RETURNS:
+*    Function:  0: Upon success. If problems:
+*
+* TODO/ISSUES/BUGS:
+*
+* LICENCE:
+*    cf. XTGeo LICENSE
```

Comment:
Something else than GPL nowadays?

Reply:
Kind of boilerplate used in all C files; could have been another text e.g. just LGPLv3. May consider it in a later refactoring.